torch/_torch_docs.py

```diff
@@ -3288,6 +3291,10 @@ def merge_dicts(*dicts):
     on both CPU and GPU; raises ``RuntimeError`` for integer division by
     zero on CPU; Integer division by zero on GPU may return any value.
 
+.. note::
+
+   Doesn't support complex inputs.
+
```

Comment:
Do we need to explicitly mention this? If yes, in that case we should also add it for remainder? (Though AFAIK, not many operators mention if they support complex dtypes)

```python
>>> import torch
>>> t = torch.tensor(1+0j)
>>> torch.fmod(t, torch.tensor(0))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
RuntimeError: "fmod_cpu" not implemented for 'ComplexFloat'
>>> torch.remainder(t, torch.tensor(0))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
RuntimeError: "remainder_cpu" not implemented for 'ComplexFloat'
```

Reply:
I think Ruberry mentioned in his comment here (https://github.com/pytorch/pytorch/pull/57941#issuecomment-837912410) for adding a note on lack of support for complex tensors. `remainder` already has it: https://pytorch.org/docs/stable/generated/torch.remainder.html. I've mimicked the comment for lack of support of complex tensors from `remainder` to `fmod` in a very recent commit.

Reply:
I think this note is already covered above

```
Supports :ref:`broadcasting to a common shape <broadcasting-semantics>`,
:ref:`type promotion <type-promotion-doc>`, and integer and float inputs.
```
which clarifies which inputs are supported.

Reply:
I think, it would be better to confirm this with @mruberry .

Reply:
It's true that the "supports statement" explains complex tensors are not supported as inputs, but the note explains why they're not supported, which is often helpful. Otherwise users might request complex support or try to add it themselves!

@anjali411, what do you think?